server/runtime_lua_nakama.go

```diff
@@ -228,9 +228,11 @@ func (n *RuntimeLuaNakamaModule) Loader(l *lua.LState) int {
 		"multi_update":                       n.multiUpdate,
 		"leaderboard_create":                 n.leaderboardCreate,
 		"leaderboard_delete":                 n.leaderboardDelete,
+		"leaderboard_list":                   n.leaderboardList,
 		"leaderboard_records_list":           n.leaderboardRecordsList,
 		"leaderboard_record_write":           n.leaderboardRecordWrite,
 		"leaderboard_record_delete":          n.leaderboardRecordDelete,
+		"leaderboardsGetId":                  n.leaderboardsGetId,
```

Comment:
Naming should be consistent with other Lua runtime functions: `leaderboardsGetId` -> `leaderboards_get_id`.